src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -183,7 +183,7 @@ internal class CommandLineParameterParser
             "file",
```

Comment:
@rjmholt, your last commit had 6 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

ConsoleHost unit tests.-Login pwsh switch.Accepts the -l switch for -Login and behaves correctly
```powershell
Expected exactly f3d556fd-5834-4676-a2a8-cfc793ace122, but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 307
307:             $result | Should -BeExactly $guid
```
ConsoleHost unit tests.-Login pwsh switch.Accepts the -L switch for -Login and behaves correctly
```powershell
Expected exactly f3d556fd-5834-4676-a2a8-cfc793ace122, but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 307
307:             $result | Should -BeExactly $guid
```
ConsoleHost unit tests.-Login pwsh switch.Accepts the -login switch for -Login and behaves correctly
```powershell
Expected exactly f3d556fd-5834-4676-a2a8-cfc793ace122, but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 307
307:             $result | Should -BeExactly $guid
```
ConsoleHost unit tests.-Login pwsh switch.Accepts the -Login switch for -Login and behaves correctly
```powershell
Expected exactly f3d556fd-5834-4676-a2a8-cfc793ace122, but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 307
307:             $result | Should -BeExactly $guid
```
ConsoleHost unit tests.-Login pwsh switch.Accepts the -LOGIN switch for -Login and behaves correctly
```powershell
Expected exactly f3d556fd-5834-4676-a2a8-cfc793ace122, but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 307
307:             $result | Should -BeExactly $guid
```